### ISSUE TITLE
terraform datadog provider : invalid or unknown key: alert_id

### ISSUE THREAD

**Issue by MBali-GitHub** [terraform-providers/terraform-provider-datadog#161]
### Terraform Version
Terraform v0.11.13
* provider.datadog: version = "~> 1.7"

### Affected Resource(s)
Datadog alert graphs

### Terraform Configuration Files
  graph {
    title = "Failed Mappings"
    viz = "alert_graph"
    alert_id = "${datadog_monitor.failedmapping.id}"
  }

I have tried a few things including adding it in the request block.

### Expected Behavior
I want to create a datadog alert graph using terraform which could reference an alert_id for an existing monitor. However, i see an error : invalid or unknown key: alert_id when trying the same.

### Actual Behavior
Error : invalid or unknown key: alert_id when trying the same.

**Comment by Veske** [comment]
You need to output the id from the monitor?

output "id" {
  value = "${datadog_monitor.monitor_name.id}"
}

**Comment by nmuesch** [comment]
Hey @MBali-GitHub Thanks for the issue report. I was able to define an alert graph for a screenboard with the following config:

```
resource "datadog_screenboard" "example_alert_graph" {
  title     = "Alert Graph Example"
  widget {
    type     = "alert_graph"
    x        = 0
    y        = 0
    title    = "alert graph title terraform"
    alert_id = "<Monitor ID>"
    viz_type = "toplist"

    time = {
      live_span = "15m"
    }
  }
}
```

Additionally, we just released version 2.0.1 of the Datadog Terraform Provider this week which includes the [Dashboard Resource](https://www.terraform.io/docs/providers/datadog/r/dashboard.html) If you want to check it out, this new resource also supports alert graphs, here is an example:

```
resource "datadog_dashboard" "alert_graph_ex_dash" {
  title         = "Alert Graph Dashboard"
  description   = "Created using the Datadog provider in Terraform"
  layout_type   = "ordered"
  is_read_only  = true

  widget {
    alert_graph_definition {
      alert_id = "<MONITOR ID>"
      viz_type = "timeseries"
      title = "Widget Title"
      time = {
        live_span = "1h"
      }
    }
  }
}
```

I'll go ahead and close this issue as this is a supported feature, but please do let me know if you continue to hit any issues when using this structure!
